assignment-1/src/app/chat-bubble/chat-bubble.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="chat-bubble" [innerHTML]="singleMessage.message" 
+[ngClass]="{'right-chat-bubble':getBubbleSide()=='right','left-chat-bubble':getBubbleSide()=='left'}"
+[ngStyle]="{backgroundColor:getBubbleSide()=='right' ? getBubbleColor():'',borderRadius:getBubbleContainerSide()}"
+
+></div>
+<div class="time"
+[ngStyle]="{textAlign: getBubbleSide()=='right' ? 'right' : 'left'}"
```

Comment:
This can be done in the style itself, i don't think we need build or listed logic for this

Reply:
External styles where handled in css instead for writing the logic in type script.